.github/workflows/update-flake-lock.yml

```
@@ -0,0 +1,66 @@
+  pull-requests: write
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: cachix/install-nix-action@v24
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)